include/openssl/fips_names.h

```diff
@@ -47,222 +38,10 @@ extern "C" {
- * performed.
- * This is disabled by default.
- * Type: OSSL_PARAM_UTF8_STRING
- */
-# define OSSL_PROV_FIPS_PARAM_RSA_PSS_SALTLEN_CHECK "rsa-pss-saltlen-check"
-
-/*
- * A boolean that determines if X9.31 padding can be used for RSA signing.
- * X9.31 RSA has been removed from FIPS 186-5, and is no longer approved for
- * signing. it may still be used for verification for legacy purposes.
- * This is disabled by default.
- * Type: OSSL_PARAM_UTF8_STRING
- */
-# define OSSL_PROV_FIPS_PARAM_RSA_SIGN_X931_PAD_DISABLED "rsa-sign-x931-pad-disabled"
-
-/*
- * A boolean that determines if the runtime FIPS key check for HKDF is
- * performed.
- * This is disabled by default.
- * Type: OSSL_PARAM_UTF8_STRING
- */
-# define OSSL_PROV_FIPS_PARAM_HKDF_KEY_CHECK "hkdf-key-check"
-
-/*
- * A boolean that determines if the runtime FIPS key check for KBKDF is
- * performed.
- * This is disabled by default.
- * Type: OSSL_PARAM_UTF8_STRING
- */
-# define OSSL_PROV_FIPS_PARAM_KBKDF_KEY_CHECK "kbkdf-key-check"
-
-/*
- * A boolean that determines if the runtime FIPS key check for TLS13 KDF is
- * performed.
- * This is disabled by default.
- * Type: OSSL_PARAM_UTF8_STRING
- */
-# define OSSL_PROV_FIPS_PARAM_TLS13_KDF_KEY_CHECK "tls13-kdf-key-check"
-
-/*
- * A boolean that determines if the runtime FIPS key check for TLS1_PRF is
- * performed.
- * This is disabled by default.
- * Type: OSSL_PARAM_UTF8_STRING
- */
-# define OSSL_PROV_FIPS_PARAM_TLS1_PRF_KEY_CHECK "tls1-prf-key-check"
-
-/*
- * A boolean that determines if the runtime FIPS key check for SSHKDF is
- * performed.
- * This is disabled by default.
- * Type: OSSL_PARAM_UTF8_STRING
- */
-# define OSSL_PROV_FIPS_PARAM_SSHKDF_KEY_CHECK "sshkdf-key-check"
-
-/*
- * A boolean that determines if the runtime FIPS key check for SSKDF is
- * performed.
- * This is disabled by default.
- * Type: OSSL_PARAM_UTF8_STRING
- */
-# define OSSL_PROV_FIPS_PARAM_SSKDF_KEY_CHECK "sskdf-key-check"
-
-/*
- * A boolean that determines if the runtime FIPS key check for X963KDF is
- * performed.
- * This is disabled by default.
- * Type: OSSL_PARAM_UTF8_STRING
- */
-# define OSSL_PROV_FIPS_PARAM_X963KDF_KEY_CHECK "x963kdf-key-check"
-
-/*
- * A boolean that determines if the runtime lower bound check for PBKDF2 is
- * performed.
- * This is enabled by default.
- * Type: OSSL_PARAM_UTF8_STRING
- */
-# define OSSL_PROV_FIPS_PARAM_PBKDF2_LOWER_BOUND_CHECK "pbkdf2-lower-bound-check"
-
-/*
- * A boolean that determines if the runtime cofactor check for ECDH is
- * performed.
- * This is disabled by default.
- * Type: OSSL_PARAM_UTF8_STRING
- */
-# define OSSL_PROV_FIPS_PARAM_ECDH_COFACTOR_CHECK "ecdh-cofactor-check"
+/* The following are provided for backwards compatability */
+# define OSSL_PROV_FIPS_PARAM_SECURITY_CHECKS OSSL_PROV_PARAM_SECURITY_CHECKS
+# define OSSL_PROV_FIPS_PARAM_TLS1_PRF_EMS_CHECK OSSL_PROV_PARAM_TLS1_PRF_EMS_CHECK
+# define OSSL_PROV_FIPS_PARAM_DRBG_TRUNC_DIGEST OSSL_PROV_PARAM_DRBG_TRUNC_DIGEST
```

Comment:
Should the names elsewhere for these be the new name?  E.g. in `fipsinstall.c` the old name is still being used.

Reply:
These are aliases, so either is fine.. We cant do much with the header as it is public..

Reply:
I figured it would be more consistent using the new throughout.